packages/server/src/fhir/search.test.ts

```diff
@@ -3397,6 +3397,27 @@ describe('FHIR Search', () => {
         expect(result.entry?.length).toBe(0);
       }));
 
+    test('Practitioner by email case insensitive', () =>
```

Comment:
Do we have a similar test for the case sensitive case matching exact casing and not the same word with different casing?

Reply:
Added one just incase